src/proposals/mips/mip-o05/MIP-O05.md

```diff
@@ -0,0 +1,81 @@
+# MIP-O05: Add weETH Market on Optimism
+
+## Summary
+
+This proposal aims to add weETH (Wrapped eETH) as a new market on Moonwell's
+Optimism deployment. weETH is the non-rebasing equivalent of eETH, a liquid
+restaking token from [Ether.fi](https://ether.fi), representing ETH staked on
+the Beacon Chain.
+
+## Proposal Details
+
+We propose adding weETH as a new collateral asset on the Moonwell Optimism
+deployment with the following details:
+
+- Asset: weETH (Wrapped eETH)
+- Address:
+  [0x5A7fACB970D094B6C7FF1df0eA68D99E6e73CBFF](https://optimistic.etherscan.io/token/0x5A7fACB970D094B6C7FF1df0eA68D99E6e73CBFF)
+- Oracle Feed:
```

Comment:
@ElliotFriedman we should also include the Chainlink contract in here, like we do on Base.

Reply:
Looks good.